pytest_catchlog.py

```diff
@@ -11,6 +12,29 @@
 
 __version__ = '1.2.0'
 
+TERMINAL = py.io.TerminalWriter(sys.stderr)  # pylint: disable=no-member
+CONSOLEHANDLER = logging.StreamHandler(TERMINAL)
+# Add the handler to logging
+logging.root.addHandler(CONSOLEHANDLER)
+# The root logging should have the lowest logging level to allow all messages
+# to be "passed" to the handlers
+logging.root.setLevel(1)
```

Comment:
This shouldn't be done at the global script level, since it effectively forbids the module to be imported by anyone, including various tools.